android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -143,6 +143,8 @@ public Bridge(
         this.config = new CapConfig(getActivity().getAssets(), config);
         Logger.init(this.config);
 
+        Splash.showOnLaunch((BridgeActivity) context, getConfig());
```

Comment:
Since it's the `Bridge` class you can call `this.config` instead of the `getConfig()`

```suggestion
        Splash.showOnLaunch((BridgeActivity) context, this.config);
```

Also, I think this should go in the 2.x branch first